otter/models/cass.py

```diff
@@ -493,11 +495,14 @@ def assemble_webhooks_in_policies(policies, webhooks):
 
 
 def verified_view(connection, view_query, del_query, data, consistency,
-                  exception_if_empty, log):
+                  exception_if_empty, log, get_deleting=False):
     """
-    Ensures the view query does not get resurrected row, i.e. one that does
-    not have "created_at" in it.  Any resurrected entry is deleted and
-    `exception_if_empty` is raised.
+    Ensures the view query on the group does not get resurrected row,
+    i.e. one that does not have "created_at" in it.  Any resurrected entry is
+    deleted and `exception_if_empty` is raised. Also raises
+    `exception_if_empty` if group's status is DELETING
+
+    :param bool get_deleting: Should it return deleting group?
 
     TODO: Should there be seperate argument for view_consistency and
```

Comment:
Non-blocking, since this wasn't touched, but this TODO seems outdated, and could probably be deleted.